src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,90 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pageNumbers = [];
+  const isPrevLinkDisabled = currentPage === 1;
+  const isNextLinkDisabled = currentPage === total;
+
+  // eslint-disable-next-line no-plusplus
+  for (let i = 1; i <= Math.ceil(total / perPage); i++) {
```

Comment:
```suggestion
  for (let i = 1; i <= Math.ceil(total / perPage); i += 1) {
```
you could write it in this way and not disable lint rule - FYI